docs/_plugins/copy_api_dirs.rb

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
You can remove this header.  We should follow up later to add Apache 2.0 headers to all files (but not ASF headers)